### DISCUSSION TITLE
Rename Wiki Page

### DISCUSSION THREAD

**Discussion by foosfam** [rfennell/AzurePipelines#992] [Q&A]
Richard,
Is there a way to rename a wiki page?

For example, I have a stage in a pipeline that creates the wiki page: RN_MySoftwareProject_QA_PROD_20210403_1

If that pipeline makes it to prod, I want to rename that page: RN_MySoftwareProject_QA_PROD_20210403_1_Released

Maybe, you don't "rename" it but copy it?  And then delete the original - Any thoughts here?  Thanks!

**Comment by rfennell** [comment]
There is no way to achieve this with the task as it stands. It is hard to see a way to implement it without becoming a very prescriptive, but I will have a think
